ci/code_checks.sh

```diff
@@ -109,7 +109,7 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     # Imports - Check formatting using isort see setup.cfg for settings
     MSG='Check import format using isort ' ; echo $MSG
-    isort --recursive --check-only pandas asv_bench
+    isort --recursive --check-only pandas asv_bench | awk -F ":" '##vso[task.logissue type=error;sourcepath=%(path)s]%(text)s'
```

Comment:
Looks good, but we just want to change the format when the environment variable `AZURE` is set. You can see how this is done for flake8. Sorry it wasn't clear in the issue.

Can you also change the order of an import to see this failing in the CI please? So we can see it working before we merge. Thanks!

Reply:
yes, thank you for being so helpful!

Reply:
Does this still grab the return code from `isort`?